test/mpi/test_alltoall.cc

```diff
@@ -60,6 +60,55 @@ TestAlltoall::TestAlltoall(size_t _msgsize, ucc_test_mpi_inplace_t _inplace,
     UCC_CHECK_SKIP(ucc_collective_init(&args, &req, team.team), test_skip);
 }
 
+TestAlltoall::TestAlltoall(size_t _msgsize, ucc_test_mpi_inplace_t _inplace,
```

Comment:
this alltoall constructor is almost identical to previous one, pls avoid code duplication

Reply:
done